src/pyOpenMS/pxds/BasicProteinInferenceAlgorithm.pxd

```diff
@@ -44,37 +46,54 @@ cdef extern from "<OpenMS/ANALYSIS/ID/BasicProteinInferenceAlgorithm.h>" namespa
         #     bpia.setParameters(p);
         #     bpia.run(peptides, proteins);
         #     #
-        #     print(proteins[0].getHits()[0].getScore()) # 0.6
-        #     print(proteins[0].getHits()[5].getScore()) # 0.9
-        #     print(proteins[0].getHits()[0].getMetaValue("nr_found_peptides")) # 1
-        #     print(proteins[0].getHits()[3].getMetaValue("nr_found_peptides")) # 2
+        #     hits = proteins[0].getHits()
+        #     print(hits[0].getScore()) # 0.6
+        #     print(hits[5].getScore()) # 0.9
+        #     print(hits[0].getMetaValue("nr_found_peptides")) # 1
+        #     print(hits[3].getMetaValue("nr_found_peptides")) # 2
```

Comment:
can this example be added to pyopenms readthedocs?

Reply:
sure. wasn't mine. Oliver introduced it. but you could just copy paste.